### PR TITLE
Task sposor edit

### PR DESCRIPTION
**Pull request by rishi9684**
#1315 Resolved (from sheet)
#739 (title added
#added sponsor edit custom form

### CODE REVIEW

File: config/templates/grids/eventSponsor-grid/actions/row-select/edit-item.vue [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    onClose() {
+      this.$refs.form.reset()
+      this.duplicateMessage = ''
+      this.dialog = false
+    },
+    async onSave() {
+      const url = this.$bitpod.getApiUrl()
+      this.formData.Type = 'Sponsor'
+      this.formData.Tags.push(this.Tags)
+      console.log('data', this.formData)
```
Comment (SukantGujar) [inline config/templates/grids/eventSponsor-grid/actions/row-select/edit-item.vue:229]: Remove

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: config/templates/grids/eventSponsor-grid/actions/row-select/edit-item.vue (hunk 1 above)
Comment (rishi9684) [reply config/templates/grids/eventSponsor-grid/actions/row-select/edit-item.vue]: Removed

File: config/templates/grids/eventSponsor-grid/actions/row-select/delete-item.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      try {
+        await this.$axios.$delete(
+          `${url}Events/${this.$route.params.id}/contacts/${id}`
+        )
+        this.snackbarText = this.$t('Messages.Success.DeletedSuccessfully', {
+          modelName: 'Contact',
+        })
+        this.snackbar = true
+      } catch (e) {
+        console.log(
```
Comment (SukantGujar) [inline config/templates/grids/eventSponsor-grid/actions/row-select/delete-item.vue:70]: error

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: config/templates/grids/eventSponsor-grid/actions/row-select/delete-item.vue (hunk 2 above)
Comment (rishi9684) [reply config/templates/grids/eventSponsor-grid/actions/row-select/delete-item.vue]: added

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**